src/apps/Hosting/HostingList.js

```diff
@@ -49,6 +50,9 @@ const HostingList = React.createClass({
 
   renderItem(item) {
     const { checkItem } = this.props;
+    const cname = _.reject(item.domains, { value: 'default' })[0];
```

Comment:
hmm I needed to add cname to an item. I tried to do it in the dialog, but then I was getting a red screen on the list ;/ could you give me any hint?